charts/consul/templates/connect-inject-clusterrole.yaml

```diff
@@ -98,6 +98,7 @@ rules:
   resources:
     - grpcroutes
     - httproutes
+    - meshgateways
```

Comment:
Will add bats tests for this before merge, but about to drop off for the weekend

Reply:
Added in https://github.com/hashicorp/consul-k8s/pull/3204/commits/59cc948afc181b5b7ccceea691ecf83939229a32